Man pages are already built in the release tarball, so we don't need latex2man.
--- a/configure.ac
+++ b/configure.ac
@@ -420,12 +420,6 @@
 AC_SUBST(DLLIB)
 AC_SUBST(BACKTRACELIB)
 
-AC_PATH_PROG([LATEX2MAN],[latex2man])
-if test "x$LATEX2MAN" = "x"; then
-  AC_MSG_WARN([latex2man not found. Install latex2man. Disabling docs.])
-  enable_documentation="no";
-fi
-
 AM_CONDITIONAL([CONFIG_DOCS], [test x$enable_documentation = xyes])
 if test "x$enable_documentation" = "xyes"; then
   AC_CONFIG_FILES(doc/Makefile doc/common.tex)
